ko/extend_your_application/README.md

```diff
@@ -52,30 +54,26 @@
 
 첫 게시물의 상세 페이지 **URL**이 http://127.0.0.1:8000/post/1/가 되게 만들 거에요.
 
-`blog/urls.py`파일에 URL을 만들어, 장고가 `post_detail` *뷰*로 보내, 게시글이 보일 수 있게 해봅시다. `url(r'^post/(?P<pk>[0-9]+)/$', views.post_detail, name='post_detail')`코드를 `blog/urls.py`파일에 추가하면 아래와 같을 보일 거에요.
+`blog/urls.py`파일에 URL을 만들어, 장고가 `post_detail` *뷰*로 보내, 게시글이 보일 수 있게 해봅시다. `path('post/<int:pk>/', views.post_detail, name='post_detail')`코드를 `blog/urls.py`파일에 추가하면 아래와 같을 보일 거에요.
```

Comment:
[Django 1.11 -> Django 2.0] Original Text:
`Let's make a URL in the blog/urls.py file to point Django to a view named post_detail, that will show an entire blog post. Open the blog/urls.py file in the code editor, and add the line path('post/<int:pk>/', views.post_detail, name='post_detail'), so that the file looks like this:`